src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -502,27 +502,46 @@ public Result get(final Get get) throws IOException {
             public Result call() throws IOException {
                 List<KeyValue> keyValueList = new ArrayList<KeyValue>();
                 byte[] family = new byte[] {};
-                ObTableClientQueryStreamResult clientQueryStreamResult;
-                ObTableQueryRequest request;
+                ObTableClientQueryAsyncStreamResult clientQueryStreamResult;
+                ObTableQueryAsyncRequest request;
                 ObTableQuery obTableQuery;
                 try {
                     if (get.getFamilyMap().keySet() == null
-                        || get.getFamilyMap().keySet().size() == 0) {
-                        obTableQuery = buildObTableQuery(get, null);
-                        request = buildObTableQueryRequest(obTableQuery,
+                            || get.getFamilyMap().keySet().isEmpty()
+                            || get.getFamilyMap().size() > 1) {
```

Comment:
Could you add some comment on the operation formats for different multi-column group operations, such as scan, get, and delete?

Reply:
done